15/umbraco-cms/reference/content-delivery-api/README.md

```diff
@@ -594,7 +594,7 @@ In this section, we will discuss some of the known limitations of the API, and h
 
 ### Protected content
 
-The Delivery API supports protected content and member authentication. This is an opt-in feature. You can read more about it in the [Protected Content in the Delivery API](protected-content-in-the-delivery-api.md) article.
+The Delivery API supports protected content and member authentication. This is an opt-in feature. You can read more about it in the [Protected Content in the Delivery API](https://docs.umbraco.com/umbraco-cms/reference/content-delivery-api/protected-content-in-the-delivery-api) article.
```

Comment:
We prefer sticking with the relative links, so I've made a suggestion that reflects that.

```suggestion
The Delivery API supports protected content and member authentication. This is an opt-in feature. You can read more about it in the [Protected Content in the Delivery API](protected-content-in-the-delivery-api/README.md) article.
```

Reply:
Sounds good to me! I found the redirection using the absolute path somewhere else in the same markdown page, If I've got the chance I'll take a look